substrate/frame/aura/Cargo.toml

```diff
@@ -50,4 +59,9 @@ try-runtime = [
 	"pallet-timestamp/try-runtime",
 	"sp-runtime/try-runtime",
 ]
+# Provide the default equivocation report system.
+default-equivocation-report-system = [
+	"pallet-authorship",
+	"pallet-session",
+]
```

Comment:
These pallets are required only if the user decides to use the offence report system which ships with substrate.

These pallets are not used otherwise.